src/routes/fetchMetricsData.ts

```diff
@@ -0,0 +1,31 @@
+import { validate, Joi } from 'express-validation';
+import UserReturnData from '../UserReturnData';
+import StatusCodes from '../StatusCodes';
+
+module.exports = (app: any, dbHandler: any) => {
+  const validation = {
+    query: Joi.object({
+      competition: Joi.string().required(),
+      team_number: Joi.string().required(),
+    }),
+  }
+  app.get('/api/fetchMetricsData', validate(validation, { keyByField: true }, {}), async (req: any, res:any) => {
+    const val: UserReturnData = new UserReturnData();
+    const { competition }: Record<string, string> = req.query;
+    const teamNumber = parseInt(req.query.team_scouted, 10);
+    val.data = await dbHandler.fetchMetricsData(req.db, competition, teamNumber).catch((e) => { console.error(e); val.err_occur = true; });
+    if (val.err_occur === false) {
+      res.json({
+        success: true,
+        competition,
+        teamNumber,
+        metrics: val.data,
```

Comment:
use `val.data.data` since overall object contains extraneous information
```suggestion
        metrics: val.data.data,
```